modules/plymouth/nixos.nix

```diff
@@ -41,6 +42,25 @@ let
     " > $themeDir/stylix.plymouth
   '';
 
+  mkPlymouthFont =
```

Comment:
Does this need to be a function?

Since it's only used once and only available within this file, it may be simpler to remove the `font` argument and reference `sansSerif` directly.

Reply:
IMO inlining the function makes it less readable, since it takes away the descriptiveness of the function name.

We should either use reference `sansSerif` directly or keep the function how it is currently.

Reply:
> IMO inlining the function makes it less readable, since it takes away the descriptiveness of the function name.

I have no strong feelings about this.

> We should either reference `sansSerif` directly

However, inlining the `sansSerif` argument might make the core logic harder to reason about.

> or keep the function how it is currently.

I suggest keeping the function as it currently is.